scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py

```diff
@@ -89,12 +89,12 @@ def get_all_sample_ids(fq_destination_table_samples, only_output_vet_tables, fq_
 
 
 def create_extract_samples_table(control_samples, fq_destination_table_samples, fq_sample_name_table,
-                                 fq_sample_mapping_table, honor_withdrawn: bool):
+                                 fq_sample_mapping_table, honor_withdrawn):
```

Comment:
I didn't understand why it was there and nowhere else, and it worked without it.

Reply:
Yeah IMHO we probably should use these [type hints](https://docs.python.org/3/library/typing.html) liberally; I'm also unsure why this was the exception rather than the rule. Perhaps something to discuss going forward for coding standards.

These type hints are not enforced by the Python runtime so code will work the same with or without them. They're intended mostly for the benefit of developers so a Python-aware IDE can warn if an actual parameter of an incompatible type is being passed.